README.md

```diff
@@ -12,7 +12,8 @@ NOTE: this project used to be called `nix-review`
 - [ofborg](https://github.com/NixOS/ofborg) support: reuses evaluation output of CI to skip local evaluation, but
   also fallbacks if ofborg is not finished
 - automatically detects target branch of pull request
-- provides a `nix-shell` with all packages, that did not fail to build
+- provides a `nix-shell` with all packages that did not fail to build
+- provides a `--batch` mode to run `nixpkgs-review` non-interactively for scripted verification
```

Comment:
Please move this to its own section called `Non-interactive execution` with an example.

Reply:
Good idea; I wrote up some basic examples in the documentation for inspiration.